xapian-applications/omega/templates/query

```diff
@@ -52,50 +52,55 @@ document.write("<span>Modified:<time datetime=\""+D+" "+T+"\"><b>"+D+"</b></time
 <body>
 <form name=P method=get
 action="$html{$or{$env{SCRIPT_NAME},omega}}" target="_top">
+<div class="query-container">
```

Comment:
Either this should be a fieldset, or you should put a fieldset inside it (wrapping `input#omega-autofocus` and `input[type=submit]`). In the latter case, this should still be a fieldset, because you can nest them.

(Alternatively, put a fieldset round the two main search controls, and don't put anything around the whole lot.)